utils/patientmatching/tests/pm_test.bal

```diff
@@ -0,0 +1,271 @@
+import ballerina/http;
```

Comment:
Add licence

Reply:
Fixed in bcec52a8981eb176c19266fb4c2e5fe367063eb6